ethcore/sync/src/chain/propagator.rs

```diff
@@ -42,6 +43,28 @@ use super::{
 	TRANSACTIONS_PACKET,
 };
 
+/// Checks if peer is able to process service transactions
+fn accepts_service_transaction(client_id: &str) -> bool {
```

Comment:
Didn't find the usage for this. What was the goal for this method?

Reply:
My bad, merged the wrong version resolving the conflict.